api/view_only_links/serializers.py

```diff
@@ -108,13 +107,13 @@ def create(self, validated_data):
         if not len(add):
             raise RelationshipPostMakesNoChanges
 
-        eligible_nodes = self.get_eligible_nodes(nodes, add)
+        eligible_nodes = self.get_eligible_nodes(nodes)
 
         for node in add:
             if not node.has_permission(user, 'admin'):
                 raise PermissionDenied
             if node not in eligible_nodes:
-                raise ValidationError(detail='The node {0} cannot be affiliated with this VOL because it is not a child of the associated VOL nodes.'.format(node._id))
+                raise ValidationError(detail='The node {0} cannot be affiliated with this VOL because it is not a child of the associated VOL node.'.format(node._id))
```

Comment:
Yay validation error! Let's go ahead and make it a tiny bit more explicit. `The node {0} cannot be affiliated with this View Only Link because the node you're trying to affiliate is not descended from the node that the View Only Link is attached to: {1}.` Unless that doesn't make any sense, in which case we can work harder at this.